.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,13 @@
-<!--- Provide a general summary of your changes in the Title above -->
-
-## Description
+<!--- Summarize your changes in the title above -->
```

Comment:
This template was part of our shared boilerplate that we agreed to use for OSS projects
https://github.com/CondeNast/opensource-check/blob/master/.github/PULL_REQUEST_TEMPLATE.md